test/holdSubject.js

```diff
@@ -25,6 +25,9 @@ describe('holdSubject', () => {
   it('should allow for adjusting bufferSize of stream', () => {
     const stream = holdSubject(3)
 
+    // Add an observer so the stream begins buffering events
+    stream.observe(() => {})
```

Comment:
Ah, right. Good to remember. I just needed `drain` the other day but hadn't used it prior.